fe/fe-core/src/main/java/org/apache/doris/PaloFe.java

```diff
@@ -173,6 +176,12 @@ public static void start(String dorisHomeDir, String pidDir, String[] args, Star
                 qeService.start();
             }
 
+            ScheduledThreadPoolExecutor refreshTimer = ThreadPoolManager.newDaemonScheduledThreadPool(1,
+                "refresh-timer-pool", true);
+            Integer refreshTime = 20;
```

Comment:
What does `20` mean? It is a magic number to me

Reply:
Indeed it pops up, it is a timer refresh time that will go after every 20 seconds to refresh the catalog that needs to be refreshed